modules/editorial-metadata/editorial-metadata.php

```diff
@@ -264,6 +302,8 @@ public function insert_editorial_metadata_term( $args ) {
 		if ( is_wp_error( $inserted_term ) ) {
 			return $inserted_term;
 		} else {
+			// Update the term with the meta_key, as we use the term_id to generate it
+			$this->update_editorial_metadata_term( $inserted_term['term_id'] );
```

Comment:
Storing the meta key with the term to make it easier to keep track of the post meta keys. The term_id isn't available until after we save it, so this weird update needs to be done but it's fast enough that it's not a problem